packages/core/src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { Theme, css } from '@emotion/react';
+import Button from '@components/Button';
+import Typography from '@components/Typography';
+import { ButtonGroupProps, Item } from './types';
+
+const ButtonItem = (theme: Theme) => css`
```

Comment:
Did you take into account responsive styles?

Reply:
Done